src/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -194,6 +195,48 @@ private static void VerifyMaxThreads(int expectedMaxw, int expectedMaxc)
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Triggers an assertion failure.")]
+        private static void SetMinThreadsTo0Test()
+        {
+            int minw, minc, maxw, maxc;
+            ThreadPool.GetMinThreads(out minw, out minc);
+            ThreadPool.GetMaxThreads(out maxw, out maxc);
+            void ResetThreadCounts()
+            {
+                Assert.True(ThreadPool.SetMaxThreads(maxw, maxc));
+                Assert.True(ThreadPool.SetMinThreads(minw, minc));
+            }
```

Comment:
Just curious why you chose to write a local function for this rather than just putting these two lines into the finally block?

Reply:
I'll change it